### PR TITLE
Settings printview

### PR DESCRIPTION
**Pull request by jh-bate**
@krystophv  see https://trello.com/c/CYlJtmZg and https://github.com/tidepool-org/blip/pull/387 for context

### CODE REVIEW

**Comment by jh-bate** [comment]
[<ux:2 dev:8 tst:2> Device settings print view in Blip](https://trello.com/c/CYlJtmZg/87-ux-2-dev-8-tst-2-device-settings-print-view-in-blip)

File: src/components/settings/NonTandem.js [hunk 1]
```diff
@@ -150,7 +164,6 @@ const NonTandem = (props) => {
         </div>
```
Comment (krystophv) [inline src/components/settings/NonTandem.js]: I think we'll need a `{renderPrintNotes()}` for the non-active schedules as well.

File: src/components/settings/settings.css [hunk 2]
```diff
@@ -89,3 +90,53 @@
...
+  .basalTable tbody tr:last-of-type {
+    font-weight: normal;
+  }
+
+  .basalHeaderBackground, .bolusHeaderBackground {
+    background-image: none;
+    color: var(--text-black);
+    font-weight: normal;
+    composes: largeSize from '../../styles/typography.css';
+  }
```
Comment (krystophv) [inline src/components/settings/settings.css]: I think could use `padding-left: 0px` here.

File: src/components/settings/NonTandem.css [hunk 3]
```diff
@@ -70,3 +71,39 @@
...
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-around;
+  }
+
+  .bolusSettingsContainer {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-around;
+  }
```
Comment (krystophv) [inline src/components/settings/NonTandem.css]: I think this and the `.basalSettingsContainer` above could both use `page-break-inside: avoid`s.

**Comment by krystophv** [comment]
This LGTM. The print views don't exactly match the design in Zeplin, so you might need to get a pass on those via a design review (I personally like the striped tables).

**Comment by krystophv** [comment]
LGTM 👍

**Pull request merged**